rq_scheduler/scheduler.py

```diff
@@ -287,13 +289,17 @@ def enqueue_job(self, job):
         interval = job.meta.get('interval', None)
         repeat = job.meta.get('repeat', None)
         cron_string = job.meta.get('cron_string', None)
+        uniq = job.meta.get('uniq', None)
 
         # If job is a repeated job, decrement counter
         if repeat:
             job.meta['repeat'] = int(repeat) - 1
 
         queue = self.get_queue_for_job(job)
-        queue.enqueue_job(job)
+        if not uniq or uniq and job.id not in queue.get_job_ids():
```

Comment:
This will kill performance if we have a queue with many jobs. I'm not against this feature, but it will have to be implemented in a scaleable way.

I think we can use a set to check for whether this specific function is already scheduled.